en/tools/config-catalog-generator/data/configs.json

```diff
@@ -1761,6 +1761,1416 @@
                 }
             ],
             "exampleFile": "indexing.toml"
-        }                
+        },
+        {
+            "title": "HTTP Transport (catalina-server)",
```

Comment:
Can we call it `HTTP Servlet Transport` ?

Reply:
will change

Reply:
resolved